inception-search-mtas/src/main/java/de/tudarmstadt/ukp/inception/search/index/mtas/KBUtility.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2017
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package de.tudarmstadt.ukp.inception.search.index.mtas;
+
+import java.util.Optional;
+
+import org.eclipse.rdf4j.model.Statement;
+import org.eclipse.rdf4j.model.ValueFactory;
+import org.eclipse.rdf4j.query.QueryEvaluationException;
+import org.eclipse.rdf4j.repository.RepositoryConnection;
+import org.eclipse.rdf4j.repository.RepositoryResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.tudarmstadt.ukp.clarin.webanno.model.Project;
+import de.tudarmstadt.ukp.clarin.webanno.support.ApplicationContextProvider;
+import de.tudarmstadt.ukp.inception.kb.KnowledgeBaseService;
+import de.tudarmstadt.ukp.inception.kb.graph.KBConcept;
+import de.tudarmstadt.ukp.inception.kb.graph.KBInstance;
+import de.tudarmstadt.ukp.inception.kb.graph.KBObject;
+import de.tudarmstadt.ukp.inception.kb.graph.RdfUtils;
+import de.tudarmstadt.ukp.inception.kb.model.KnowledgeBase;
+
+public class KBUtility {
+
+    private final Logger log = LoggerFactory.getLogger(getClass());
+    private KnowledgeBaseService kbService;
+    public KBUtility() {
+        kbService = ApplicationContextProvider.getApplicationContext()
+                .getBean(KnowledgeBaseService.class);
+    }
+
+    /**
+     * Return Class type of KBObject
+     * 
+     * @param kbObject
+     * @return
+     */
+    public Optional<String> getType(Optional<KBObject> kbObject) {
+        return kbObject.map((p) -> p.getClass().getSimpleName());
+    }
+
+    /**
+     * Read identifier URI and return Optional<KBObject>
+     * @param aProject
+     * @param aIdentifier
+     * @return
+     */
+    public Optional<KBObject> readKBIdentifier(Project aProject, String aIdentifier) {
```

Comment:
Once https://github.com/inception-project/inception/pull/273 is merged, this can and should be done using `ConceptFeatureSupport.wrapFeatureValue`. Instead of using `ConceptFeatureSupport.wrapFeatureValue` directly, the appropriate feature support for a given feature should be obtained from the `FeatureSupportRegistry` and then `wrapFeatureValue` should be called.